MarkBind/vue-strap
# Enable Pic component to be responsive

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [x] Enhancement to an existing feature


Part of MarkBind/markbind#335



**What is the rationale for this request?**
Sex

**What changes did you make? (Give an overview)**
- Remove height attribute
- Add `img-fluid` responsive class and `img-rounded` class

Before | After
--- | ---
![responsive-image](https://user-images.githubusercontent.com/31084833/42675924-96592860-86a8-11e8-922a-06abbda26f84.PNG) | ![335-pic-after](https://user-images.githubusercontent.com/31084833/42806585-5bbe94ce-89e1-11e8-9784-52067026ed96.PNG) |

TODO |
--- |
Update user guide with reference to the deprecated `height` attribute |


**Testing instructions:**
- Run `npm run build` and paste the updated `vue-strap.min.js` into MarkBind's asset folder.
- Run `markbind serve docs`
- Experiment by altering the `width` of the img at Pic section while changing window sizes.
- The image should never overflow outside the container, and never exceeds its parsed `width`.

```html
<img src="https://vuejs.org/images/logo.png" alt="Logo" width="300">
```
